api/integrations/github/client.py

```diff
@@ -12,3 +12,12 @@ def generate_token(installation_id: str, app_id: int) -> str:
     Github(auth=auth)
     token = auth.token
     return token
+
+
+def generate_jwt_token(app_id: int) -> str:  # pragma: no cover
```

Comment:
Naming of this function could be better. `generate` assumes Core API generates something on its end.

```suggestion
def get_jwt_token(app_id: int) -> str:  # pragma: no cover
```

Reply:
In this context is valid since we are inside the integrations.github App and in its client module.